docs/conductor/getting-started.md

```diff
@@ -31,13 +38,13 @@ We recommend to create a dedicated authentication token with update access. You
 
 ![Create Authentication Token](/Resources/public/img/docs/conductor/authentication-token.png)
 
-The contents of the variable should look like 
+The contents of the variable should look like
 
 ```json
 {"http-basic": {"repo.packagist.com": {"username": "token", "password": "packagist_out_73a81c..." }}}
 ```
 
-Conductor needs to verify your CI setup before you can start receiving pull requests.
+Conductor needs to verify your CI setup before you can start receiving PRs.
 
 ## Verify your CI setup
```

Comment:
I think there this needs a sentence, that you go to the Conductor tab and click the name of the package of your repository you just added the workflow to.

Reply:
moved